SETUP/lint_php_files.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -euo pipefail
```

Comment:
This needs to be after line 13 or `./lint_php_files.sh` fails because `$1` isn't set.

Reply:
Or we can use BASHisms for handling possibly unbound variables. Done!